hangupsbot/handlers.py

```diff
@@ -167,7 +167,10 @@ def handle_command(self, event):
 
         # check that a bot alias is used e.g. /bot
         if not event.text.split()[0].lower() in self.bot_command:
-            return
+            if self.bot.conversations.catalog[event.conv_id]["type"] == "ONE_TO_ONE" and self.bot.get_config_option('auto_alias_one_to_one'):
```

Comment:
If *auto_alias_one_to_one* isn't present in the config file, `get_config_option` returns None which functions as false and this enhancement has no effect on existing users.